### DISCUSSION TITLE
Multiple .ledger files

### DISCUSSION THREAD

**Discussion by AnandChowdhary** [ananthakumaran/paisa#99] [Ideas]
I have a few types of accounts and it's becoming hard to track them all in one giant ledger file for using with Paisa. It would be amazing if I can import many different .ledger files in the app, and it just treats them as one.

As an alternate option, I can have different files on my system and `cat` them together, and then import then to Paisa.

**Comment by ananthakumaran** [comment]
`include` is what you are looking for, check the  [journal](https://paisa.fyi/reference/journal/) docs. I personally have more than 20 files.

**Reply by AnandChowdhary** [reply]
Thank you!
